lib/Dialect/TTIR/IR/TTIROps.cpp

```diff
@@ -27,6 +27,24 @@
 #define GET_OP_CLASSES
 #include "ttmlir/Dialect/TTIR/IR/TTIROps.cpp.inc"
 
+//===----------------------------------------------------------------------===//
+// ClampOp
+//===----------------------------------------------------------------------===//
+
+::mlir::LogicalResult mlir::tt::ttir::ClampOp::verify() {
```

Comment:
Please add negative test cases to cover all possible failures for TTIR and TTNN verifiers. You can find the following examples:
https://github.com/tenstorrent/tt-mlir/blob/main/test/ttmlir/Dialect/TTIR/matmul/matmul_tests_negative.mlir
https://github.com/tenstorrent/tt-mlir/blob/main/test/ttmlir/Dialect/TTNN/matmul/matmul_tests_negative.mlir

Reply:
thanks. negative tests added.